configuration/ConfigHandler.cpp

```diff
@@ -0,0 +1,50 @@
+#include "ConfigHandler.hpp"
+#include <iostream>
+#include <fstream>
+#include "../yaml-cpp/include/yaml-cpp/yaml.h"
+
+Configuration::ConfigHandler::ConfigHandler(std::string filePath)
+{
+    this->getConfigFile(filePath);
+}
+
+Configuration::ConfigHandler::~ConfigHandler(){}
+
+void Configuration::ConfigHandler::getConfigFile(std::string configFile)
+{
+    YAML::Node config = YAML::LoadFile(configFile);
+
+    const std::string ip = config["network"]["ip"].as<std::string>();
+    const int port = config["network"]["port"].as<int>();
+    const int max_players = config["general"]["max_players"].as<int>();
+    const std::string motd = config["general"]["motd"].as<std::string>();
+
+    this->_configFile.insert(std::pair<std::string, std::string>("ip", ip));
```

Comment:
Please simplify this by using initializer list instead of creating a whole `std::pair` like this : 
```cpp
this->_configFile.insert({"ip", ip});
```